src/dgipy/dgidb.py

```diff
@@ -220,6 +220,75 @@ def get_drug_applications(
     return result
 
 
+def get_clinical_trials(
+    terms: str | list,
+) -> pd.DataFrame:  # TODO: Better error handling for new_row?, use_pandas=False
+    """Perform a look up for clinical trials data for drug or drugs of interest
+
+    :param terms: drug or drugs of interest
+    :return: all clinical trials data for drugs of interest in a DataFrame
+    """
+    base_url = "https://clinicaltrials.gov/api/v2/studies?format=json"
+    rows_list = []
+
+    if isinstance(terms, str):
+        terms = [terms]
+
+    for drug in terms:
+        intr_url = f"&query.intr={drug}"
+        full_uri = base_url + intr_url  # TODO: + cond_url + term_url
+        try:
+            r = requests.get(full_uri, timeout=20)
+        except requests.exceptions.Timeout:
+            print(f"Timeout occured for {drug}")  # noqa: T201
+        except requests.exceptions.RequestException as e:
+            print(f"Request exception for {drug}: {e}")  # noqa: T201
```

Comment:
Also these should raise actual errors and log the issue, you generally shouldn't be making `print` calls in a library ever